### PR TITLE
Add bow_encoder for imports

### PR DESCRIPTION
**Pull request by nkconnor**

### CODE REVIEW

**Comment by nkconnor** [comment]
I signed the CLA

**Comment by nkconnor** [comment]
importing `tensorflow.contrib.layers.bow_encoder` fails with a "couldn't find attribute bow_encoder on module" type error, and so I have been importing `tensorflow.contrib.layers.python.layers.encoders.bow_encoder`.

It is my understanding from [another PR/issue](https://github.com/tensorflow/tensorflow/issues/7569) that this commit will fix the imports.

File: tensorflow/contrib/layers/__init__.py [hunk 1]
```diff
@@ -82,6 +82,8 @@
 @@weighted_sum_from_feature_columns
 @@infer_real_valued_columns
 @@sequence_input_from_feature_columns
+
+@@bow_encoder
```
Comment (jhseu) [inline tensorflow/contrib/layers/__init__.py]: nit: can you move above next to embed_sequence?

**Review by jhseu — APPROVED** [review]

**Comment by jhseu** [comment]
Jenkins, test this please

**Comment by jhseu** [comment]
Jenkins, test this please

**Pull request merged**